### PR TITLE
LSR-862 Allow on demand rendering of the screenlet view

### PR DESCRIPTION
**Pull request by nhpatt**
This would be how it would be used:

```
UserPortraitScreenlet userPortraitScreenlet = new UserPortraitScreenlet(getContext());
userPortraitScreenlet.setLayoutParams(new LayoutParams(getLayoutParams().width, getLayoutParams().height));
userPortraitScreenlet.setAutoLoad(true);
userPortraitScreenlet.setOfflinePolicy(OfflinePolicy.REMOTE_FIRST);
userPortraitScreenlet.setUserId(20199);
userPortraitScreenlet.render(R.layout.userportrait_default);
```

@dgarciasarai

### CODE REVIEW

File: android/library/src/main/java/com/liferay/mobile/screens/base/BaseScreenlet.java [hunk 1]
```diff
@@ -120,7 +130,17 @@ protected I prepareInteractor(String actionName) {
 	protected void init(Context context, AttributeSet attributes) {
 		LiferayScreensContext.init(context);
 
-		_screenletView = createScreenletView(context, attributes);
+		assignView(createScreenletView(context, attributes));
+	}
+
+	public void render(int layoutId) {
```
Comment (jmnavarro) [inline android/library/src/main/java/com/liferay/mobile/screens/base/BaseScreenlet.java]: do we need to change existing screenlets to use this new `render` method?

Comment (nhpatt) [inline android/library/src/main/java/com/liferay/mobile/screens/base/BaseScreenlet.java]: No, it's just for the use case where you want to dynamically create a screenlet from code.

Comment (jmnavarro) [inline android/library/src/main/java/com/liferay/mobile/screens/base/BaseScreenlet.java]: 👍

**Comment by jmnavarro** [comment]
👍

**Pull request merged**